signing_service/signing_service/signing_service.py

```diff
@@ -570,6 +583,15 @@ def _parse_arguments() -> argparse.Namespace:
     )
     crash_logger.handlers[0].client = raven_client  # type: ignore
 
+    if any('smtp-notifier' in args for args in argparse.ArgumentParser().parse_known_args()):  # type: ignore
+        notifier_class = EmailNotifier(
```

Comment:
```suggestion
        notifier = EmailNotifier(
```

Reply:
pylint is crying about that:
`************* Module signing_service.signing_service
W: 96, 8: Redefining name 'notifier' from outer scope (line 587) (redefined-outer-name)
`
Maybe `notifier_type` if not `notifier_class`

Reply:
just make a `main` function and then it will become local scope of it.

Reply:
Done